astropy/io/ascii/tests/test_ecsv.py

```diff
@@ -35,7 +35,7 @@
 DTYPES = ['bool', 'int8', 'int16', 'int32', 'int64', 'uint8', 'uint16', 'uint32',
           'uint64', 'float16', 'float32', 'float64', 'float128',
           'str']
-if os.name == 'nt' or sys.maxsize <= 2**32:
+if not hasattr(np, 'float128') or os.name == 'nt' or sys.maxsize <= 2**32:
```

Comment:
@mhvk - as our resident Numpy expert, is the first clause of the if statement sufficient?

Reply:
Indeed, I would guess that if `np.float128` exists, it is also more or less functional! But the only way to make sure may be to just try...

Reply:
On the other hand, if it works, why break it :laughing: - maybe we can just merge as-is

Reply:
I'll just merge but if anyone feels strongly feel free to open a follow-up!